cli/src/generate/render.rs

```diff
@@ -867,6 +878,7 @@ impl Generator {
         ranges: &[Range<char>],
         is_included: bool,
         indent_count: usize,
+        skip_eof: bool,
```

Comment:
I think it would be correct to *always* add the `!eof &&` clause if the character range contains the null character. It seems like we shouldn't ever be "advancing" if we're at EOF.